database-commons/src/main/java/io/cdap/plugin/db/batch/sink/ETLDBOutputFormat.java

```diff
@@ -110,9 +110,14 @@ public void close(TaskAttemptContext context) throws IOException {
         }
 
         @Override
-        public void write(K key, V value) throws IOException {
-          super.write(key, value);
+        public void write(K key, V value) {
           emptyData = false;
+          try {
+            key.write(getStatement());
+            getStatement().addBatch();
+          } catch (SQLException e) {
+            LOG.warn("Failed to write value to database", e);
```

Comment:
is there a reason it logs and continues rather than throwing an exception and failing immediately?

Reply:
it just copies logic from parent class. Did not want to break something on which DBOutputFormat relies.